x-pack/plugins/security_solution/server/lib/detection_engine/rules/prepackaged_rules/credential_access_lsass_memdump_file_created.json

```diff
@@ -56,5 +56,5 @@
   "timeline_title": "Comprehensive File Timeline",
   "timestamp_override": "event.ingested",
   "type": "eql",
-  "version": 6
+  "version": 7
```

Comment:
This looks to be an unnecessary version bump? Was it missed in the previous batch of `8.2` updates? Sounds like a bug somewhere... 🤔